### PR TITLE
Enumerate new_results_not_in_new_initial_negatives

### PR DESCRIPTION
**Pull request by pbchase**
This code that leads me toward better code.  I am using it now to get better data out of `write_lab_results_to_results_project.R` until we automate the movement of that same data,

Add function `get_new_results_not_in_new_initial_negatives`.
Use `get_new_results_not_in_new_initial_negatives` in `write_lab_results_to_results_project.R`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,8 +54,7 @@ This suggests the following timing:
  - `load_person_data_into_results.R` - run automatically at 1210 and 1710
  - `write_lab_results_to_results_project.R` - run manually at 0800 and 1215
  - `compare_contact_tracing_records_to_results.R` - run manually at 0805 and 1220
- - `send_cleared_status_to_person_hub.R` - run manually at 0850 and 1250
- - `send_cleared_status_to_person_hub.R` - run automatically, 50 minutes past the hour at 0,4,16,20
+ - `send_cleared_status_to_person_hub.R` - run automatically, 50 minutes past the hour at 0,4,8.12,16,20
```
Comment (ChemiKyle) [inline README.md:57]:
```suggestion
 - `send_cleared_status_to_person_hub.R` - run automatically, 50 minutes past the hour at 0,4,8,12,16,20
```
Typo fix . -> ,

**Comment by ChemiKyle** [comment]
I saw you make this PR around 11am yesterday and didn't realize you'd requested my review

**Comment by pbchase** [comment]
> I saw you make this PR around 11am yesterday and didn't realize you'd requested my review

Initially I forgot to request you, so that one's definitely on me.

**Comment by ChemiKyle** [comment]
Why the code duplication?  
https://github.com/ctsit/tt_engine/pull/49/files#diff-9bb0111639f96b5c0b7643b7eff9907dR329-R337

It seems the same dataframe could be achieved by just the following (a `new_initial_negatives` df with fewer columns already exists by the time this function is called)
```R
get_new_results_not_in_new_initial_negatives <- function(new_lab_results, new_initial_negatives) {
  new_results_not_in_new_initial_negatives <- new_lab_results %>%
    anti_join(new_initial_negatives, by = 'record_id')
  return(new_results_not_in_new_initial_negatives)
}
```
 
I feel like I'm missing something critical in the function that creates `new_initial_negatives`.

https://github.com/ctsit/tt_engine/blob/a0314282792134562990420139f8053bc3b927a7/functions.R#L306-L325

**Comment by pbchase** [comment]
Duplicated code removed

**Pull request merged**